### PR TITLE
Report back if we can't start the plugin.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

**Comment by nomis52** [comment]
2nd last change for 0.9.6 I think

File: olad/OlaServerServiceImpl.cpp [hunk 1]
```diff
@@ -497,7 +497,9 @@ void OlaServerServiceImpl::SetPluginState(
     OLA_DEBUG << "SetPluginState to " << request->enabled()
               << " for plugin " << plugin->Name();
     if (request->enabled()) {
-      m_plugin_manager->EnableAndStartPlugin(plugin_id);
+      if (!m_plugin_manager->EnableAndStartPlugin(plugin_id)) {
+        controller->SetFailed("Failed to start plugin");
```
Comment (peternewman) [inline olad/OlaServerServiceImpl.cpp]: Give the plugin name.in the error message.

**Comment by peternewman** [comment]
LGTM

File: olad/OlaServerServiceImpl.cpp (hunk 1 above)
Comment (nomis52) [inline olad/OlaServerServiceImpl.cpp]: Done

**Pull request merged**